src/components/block-tunes/block-tune-move-up.ts

```diff
@@ -3,9 +3,9 @@
  * @classdesc Editor's default tune that moves up selected block
  * @copyright <CodeX Team> 2018
  */
-import { API, BlockTune } from '../../../types';
-import { IconChevronUp } from '@codexteam/icons';
-import { TunesMenuConfig } from '../../../types/tools';
+import { API, BlockTune } from "../../../types";
```

Comment:
Thanks for the PR, please follow our Code Style

Reply:
Oops, prettier kicked in. I Resolved it. Thanks.